packages/quantification/src/__tests__/net-quantification.test.ts

```diff
@@ -0,0 +1,140 @@
+import { getNetQuantificationProjection } from '../net-quantification';
+
+/**
+ * If you'd like to see the output from the algorithm, add `console` as the second
+ * parameter to `getNetQuantificationProjection`
+ */
+
+it('should return a total of 18 NRT', () => {
+  const testData = [
+    {
+      somscAnnualDifferencesBetweenFutureAndBaselineScenarios: {
+        '2016': -30,
+        '2015': 10,
+        '2017': 9,
+        '2014': 0,
+      },
+    },
+    {
+      somscAnnualDifferencesBetweenFutureAndBaselineScenarios: {
+        '2016': 10,
+        '2017': 9,
+        '2018': 0,
+        '2015': 10,
+      },
+    },
+  ];
+
+  expect(getNetQuantificationProjection(testData)).toStrictEqual([
+    {
+      '2014': 0,
+      '2015': 0,
+      '2016': 0,
+      '2017': 0,
+      '2018': 0,
+    },
+    {
+      '2014': 0,
+      '2015': 10,
```

Comment:
This is probably me just not understanding how the algo is supposed to work, but it does surprise me that negative years would roll up into the totals from future years rather than past years. Like it makes more sense to me that an emission would offset the removals of the past rather than the future (and that would also mean we would be selling "fresher" carbon removals from more recent years). Totally get it if that's just not how it's supposed to work for whatever reason, but a little curious about the why. That's maybe more of a question for the supply team though rather than PR review ;)

Reply:
Just to clarify what I mean here, in the results I would anticipate that the 2015 tonnes would be 0, and that 10 tonnes would be otherwise accounted for in 2016/2017. I can maybe draw out a diagram if it's still not clear what I'm getting at.

Reply:
Yeah, the table I provided in the comment was verified by Jaycen and Rebekah. I'm not familiar with the reasoning behind it but I believe the implementation is correct.